File: src/main/java/fr/marstech/mtlinkspray/repository/AbuseReportRepository.java
package fr.marstech.mtlinkspray.repository;

import fr.marstech.mtlinkspray.entity.AbuseReportEntity;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AbuseReportRepository extends MongoRepository<AbuseReportEntity, String> {

}

File: src/main/kotlin/fr/marstech/mtlinkspray/repository/AbuseReportRepository.kt
package fr.marstech.mtlinkspray.repository

import fr.marstech.mtlinkspray.entity.AbuseReportEntity
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface AbuseReportRepository : MongoRepository<AbuseReportEntity, String>
